spec/views/repo_spec.rb

```diff
@@ -2,17 +2,20 @@
 
 describe 'repository/show.html.erb', type: :view do
   let(:repo) do
-    repo = Repository.new('owner_name', 'repo_name')
+    repo = Repository.from_owner_and_name('owner_name', 'repo_name')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.